Spanner/tests/System/QueryTest.php

```diff
@@ -169,26 +149,6 @@ public function testBindInt64ParameterWithInt64Class()
         $this->assertEquals(1337, $row['foo']);
     }
 
-    /**
-     * covers 26
-     */
-    public function testBindNullIntParameter()
-    {
-        $db = self::$database;
-
-        $res = $db->execute('SELECT @param as foo', [
-            'parameters' => [
-                'param' => null
-            ],
-            'types' => [
-                'param' => Database::TYPE_INT64
-            ]
-        ]);
-
-        $row = $res->rows()->current();
-        $this->assertNull($row['foo']);
-    }
-
     /**
      * covers 27
      */
```

Comment:
Please add a test in GoogleSQL dialect for float32, similar to `testBindFloat64Parameter()`

Reply:
Fixed it in c260a8603b46d5aeb0fa42ee18a810e1f175612a